### PR TITLE
Added in error handling for an invalid formatted private key string. …

### PR DESCRIPTION
**Pull request by joelwass**
…The crypto error handling for this error isnt intuitive, and it should be caught before being passed along to crypto

### CODE REVIEW

**Comment by joelwass** [comment]
status of this?

**Comment by joelwass** [comment]
bump

File: lib/cloudfrontUtil.js [hunk 1]
```diff
@@ -214,6 +214,11 @@ function _getPrivateKey(params) {
     privateKeyString = pem.toString('ascii');
   }
 
+  var lineBreakExists = /\r|\n/.exec(privateKeyString);
+  if (!lineBreakExists) {
```
Comment (jasonsims) [inline lib/cloudfrontUtil.js]: Since we're only interested in whether or not the regex found a match, can you use `.test` instead?

Comment (joelwass) [inline lib/cloudfrontUtil.js]: duly noted and fixed

**Comment by joelwass** [comment]
test added - let me know what you think. i tried to conform to your methods

**Pull request merged**